content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+
+#### Translating proxy requests and proxy entities
+
+See the [guide to monitoring external resources][47] to re-configure `proxy_requests` attributes and update your proxy check configuration, and see the [entity reference][6] to re-create your proxy client configurations as Sensu Go proxy entities.
+
+#### Translating hooks
+
+Check hooks are now a resource type in Sensu Go, meaning that hooks can be created, managed, and reused independently of check definitions.
+You can also execute multiple hooks for any given response code.
+See the [guide][55] and [hooks reference docs][8] to re-create your Sensu 1.x hooks as Sensu Go hook resources. 
+
+#### Custom attributes
+
+Custom check attributes are no longer supported in Sensu Go.
+Instead, Sensu Go provides the ability to add custom labels and annotations to entities, checks, assets, hooks, filters, mutators, handlers, and silences.
+The translator stores all check extended attributes in the check metadata annotation named `sensu.io.json_attributes`. 
+See the [check reference][56] for more information about using labels and annotations in check definitions.
+
+#### Check plugins
+
+Within the Sensu Plugins org, see individual plugin READMEs for compatibility status with Sensu Go.
+If you're doing a side-by-side migration, Sensu-Go-compatible plugins installed outside of opt/sensu will work with Sensu Go agents.
+Otherwise, you can re-install your check plugins onto your Sensu agent nodes using `sensu-install`; see the [guide][22] to installing the `sensu-install` tool for use with Sensu Go.
+
+Although not required to run Sensu Go, we recommend installing plugins with [assets](#activate-assets) where possible.
+
+### 3. Translate filters
+
+Ruby eval logic used in Sensu 1.x filters has been replaced with JavaScript expressions in Sensu Go, opening up powerful possibilities to filter events based on occurrences and other event attributes.
+As a result, you'll need to rewrite your Sensu 1.x filters in Sensu Go format.
+
+First, review your 1.x handlers to see which filters are being used. Then, using the [filter reference][57] and [guide to using filters][58], re-write your filters using Sensu Go expressions. Also check out the [blog post][59] for a deep dive into Sensu Go filter capabilities.
+
+{{< highlight shell >}}
+# Sensu 1.x hourly filter
+{
+  "filters": {
+    "recurrences": {
+      "attributes": {
+        "occurrences": "eval: value == 1 || value % 60 == 0"
+      }
+    }
+  }
+}
+
+# Sensu Go hourly filter
+  {
+    "metadata": {
+      "name": "hourly",
+      "namespace": "default"
+    },
+    "action": "allow",
+    "expressions": [
+      "event.check.occurrences == 1 || event.check.occurrences % (3600 / event.check.interval) == 0"
+    ],
+    "runtime_assets": null
+  }
+{{< /highlight >}}
+
+### 4. Translate handlers

```

Comment:
do we want to drop a note about the sensu-plugin gem providing a helpful mutator that will map from Go events into Core events?  This can be used to help bridge the gap until handlers are updated for Sensu Go native events.